README.md

```
@@ -1,58 +1,319 @@
-## Project Description (QR Code-Based Metro Ticketing System)
-### Introduction 
+## 👥 Team Information: NetNinjas
+
+| Role | Name | GitHub |
+|------|------|--------|
+| **Mentor** | Jaber Kibria | [![GitHub](https://img.shields.io/badge/GitHub-100000?style=flat&logo=github&logoColor=white)](https://github.com/mhsjaber) |
+| **Team Member** | Md Nazmul Hossain | [![GitHub](https://img.shields.io/badge/GitHub-100000?style=flat&logo=github&logoColor=white)](https://github.com/nazmulhossin) |
+|                 | Sujan Roy | [![GitHub](https://img.shields.io/badge/GitHub-100000?style=flat&logo=github&logoColor=white)](https://github.com/sujan-roy24) |
+|                 | Rabbani Islam Refat | [![GitHub](https://img.shields.io/badge/GitHub-100000?style=flat&logo=github&logoColor=white)](https://github.com/refat75) |
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)